src/app/app-routing.module.ts

```diff
@@ -39,6 +40,47 @@ export const routes: Routes = [
         path: ProjectLinks.URL_PAGE_NOT_FOUND,
         component: PageNotFoundComponent,
     },
+    {
+        path: ProjectLinks.URL_SETTINGS,
```

Comment:
Can the children be routed in the same style as we have in the dataset view? Via query parameters rather than custom URL parts. We direct all these to the same component anyway.

Reply:
Also, these identifiers should become constants in `ProjectLinks`

Reply:
I think this URL also requires authenticaion guard. If it is attempted when user is not logged in, we are currently displaying empty settings page. This should redirect to home instead. Please implement a guard class, which we will use for all pages that require a logged in user.